bcos-framework/bcos-framework/ledger/LedgerTypeDef.h

```diff
@@ -69,4 +69,5 @@ constexpr static std::string_view SYS_HASH_2_RECEIPT{"s_hash_2_receipt"};
 constexpr static std::string_view DAG_TRANSFER{"/tables/dag_transfer"};
 constexpr static std::string_view SMALLBANK_TRANSFER{"/tables/smallbank_transfer"};
 constexpr static std::string_view SYS_CODE_BINARY{"s_code_binary"};
+constexpr static std::string_view SYS_CODE_ABI{"s_code_abi"};
```

Comment:
```suggestion
constexpr static std::string_view SYS_CODE_ABI{"s_contract_abi"};
```
建议用contract替换code更准确一点

Reply:
是的

Reply:
已修改